### PR TITLE
Fix asset row rendering issue when an asset is missing an icon

### PR DESCRIPTION
**Pull request by danielmkm**
# Description

When an asset is missing an icon, the rendering of the "My Wallet" component breaks down.

Screenshot 2021-11-16 at 07 19 38

Apply a `display: flex` to the row so that it properly inlines the div component that gets rendered when the asset has no icon image.

Tested the fix with assets only with icons, a mixture, and all without. Every combination renders properly now.

Screenshot 2021-11-16 at 07 38 14

Screenshot 2021-11-16 at 07 42 02

Screenshot 2021-11-16 at 07 40 07

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## How should this be tested?

- [ ] Have an asset in your wallet that does not have an icon, navigate to the trade page, ensure the items are still laid out correctly 
- [ ] Ensure that assets with images still render correctly

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] My changes generate no new console warnings
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Review by garethfuller — APPROVED** [review]
Thanks again @danielmkm! @1saf was going to look into this after pool creation so I'll let him review first before merging. One thing I'm not sure about is if the scope of BalAssetSet should consider rows at all. I originally built this component and it was only supposed to handle the stacked horizontal asset images we have in the pool tables.

**Review by 1saf — APPROVED** [review]

**Comment by 1saf** [comment]
Thanks for this

**Pull request merged**
